swagger/swagger.yaml

```diff
@@ -1381,6 +1382,12 @@ parameters:
     description: ensures that all files and directories are part of intended projects.
     required: false
     type: string
+  actor_filter:
+    name: actor_filter
+    in: query
+    description: includes filtering according to the name of creator, curator, publisher or rights_holder. The name can be organisation or person. Multiple actor_filters can be applied simultaneously (AND) or separately (OR) by using 'condition_separator'. Default separator is AND. Examples 'creator_person=person_name', 'publisher_organization=some organisation&creator_person=person_name&condition_separator=or'. 
```

Comment:
Could use some formatting, and making clear that the actual query parameter name is not "actor_filter":

``
description: |
  actor_filters are a collection of filter parameters for filtering according to the name
  of creator, curator, publisher or rights_holder actors. Actor type must be defined as
  a suffix in the filter name ('_person' or '_organization'). Multiple actor_filters can
  be applied simultaneously (AND) or separately (OR) by using 'condition_separator'.
  Default separator is AND.

  Examples\&\#58;

  '?creator_person=person_name'
  '?publisher_organization=some organisation&creator_person=person_name&condition_separator=or'
``